law-tayh-3al-mariiii55/MinHeap.cpp

```diff
@@ -0,0 +1,82 @@
+#include"MinHeap.h"
+MinHeap::MinHeap(int c)
+{
+	Heap_size = 0;
+	capacity = c;
+	ptr = new int[c];
+}
+
+void MinHeap::InsertKey(int k)
+{
+	if (Heap_size == capacity)
+	{
+		cout << " Could not insert a Key" << endl;
```

Comment:
It will be good if we can change the size dynamically which means even if we reach the max number of elements in the heap, we can still add elements. For example we can create a new heap with a size larger than the current heap and copy all the old elements in the new heap and then insert the new one.
Also it well be better if you throw an exception and not just output an error message.